bayesian/utility/user_utils.py

```diff
@@ -57,61 +61,28 @@ def create_or_update_user(user_id, snyk_api_token, user_source):
         raise UserException("Error updating user") from e
 
 
-def create_or_update_user_in_cache(user_id, snyk_api_token, user_source):
+def create_or_update_user_in_cache(user_id, snyk_api_token):
     """Create or Update User in cache."""
     if ENABLE_USER_CACHING:
-        db_cache_file_path = DB_CACHE_DIR + "/" + user_id + ".json"
         user_detail = {"user_id": user_id,
-                       "user_source": user_source,
                        "snyk_api_token": snyk_api_token,
-                       "status": UserStatus.REGISTERED.name,
-                       "created_date": datetime.datetime.now(),
-                       "registered_date": datetime.datetime.now()}
+                       "status": UserStatus.REGISTERED.name}
         try:
-            with open(db_cache_file_path, 'w', encoding='utf-8') as file:
-                json.dump(user_detail, file, ensure_ascii=False, indent=4, default=str)
-
-            logger.info("User added in cache with id %s", user_id)
+            url = _JOB_API_URL + "/create_or_update_user_in_cache"
```

Comment:
check to see if caching is enabled should be done in api server itself. it will save us the api call

Reply:
Yes in L66 you can see if ENABLE_USER_CACHING is False it wont go inside and wont call jobs API.